### PR TITLE
Drag handler for LocalParticipant component is changed.

### PR DESCRIPTION
**Pull request by hasevr**
To scroll even when mouse cursor is stopped on the edge of the screen, I need to use timer call back.
In timer call back, I can use only old state when I use useState().
However, I need some state variables which can referred and updated from onTimer().

Therefore I added MemoryById class and memoObject() function in the "memoObject.ts".
This is not work completely. I need some key for each component. But I wan unable to find such key from a function component of React. I use return variable of useState(). However, they are different between the first call and calls later. So the two static objects are created.
Usually useState() is called more than twice and it works. An unused object is allocated for the first call.

### CODE REVIEW

**Comment by hasevr** [comment]
I found I did not understand the usage of useRef() to keep state of React.FC component. Unlink useState(), useRef() can keep mutable object and the object can be modified any time.
So, I don't need original one. Just use useRef() for the purpose.

**Comment by zlguo1996** [comment]
Sorry I could not understand the purpose: 'To scroll even when mouse cursor is stopped on the edge of the screen'.
1. Do you mean scroll to zoom the map?
2. Why mouse cursor would be stopped? I thought user could freely move their mouse wherever it is.
@hasevr

**Comment by hasevr** [comment]
> Do you mean scroll to zoom the map?
No move map translationally.

> Why mouse cursor would be stopped? I thought user could freely move their mouse wherever it is.
Because the range of mouse cursor is limited inside the window. Once avatar reach to the edge, avatar will stop.
In such case, the map will scroll and avatar can continuously move.

File: services/conference/src/scripts/components/utils/KeyHandler.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import {useEffect, useRef} from 'react'
+
+export class KeyHandlerReact<ET extends Element>{
```
Comment (zlguo1996) [inline services/conference/src/scripts/components/utils/KeyHandler.ts:3]: It seems both KeyHandlerReact and KeyHandlerPlain are not used.

**Review by zlguo1996 — APPROVED** [review]
Thank you @hasevr . The code is good. Two small things:
- one file (KeyHandler.ts) seems unused
- layout of echo cancel fab is a bit strange
image

File: services/conference/src/scripts/components/utils/KeyHandler.ts (hunk 1 above)
Comment (hasevr) [reply services/conference/src/scripts/components/utils/KeyHandler.ts:3]: Sorry I've used these function later. But included in this version.

Comment (hasevr) [reply services/conference/src/scripts/components/utils/KeyHandler.ts:3]: Thank you for finding bug of Fab icons. I'll fix them soon.

**Pull request merged**